BaseTools/.gitignore

```diff
@@ -17,5 +17,6 @@ Source/C/VfrCompile/VfrTokens.h
 Source/C/bin/
 Source/C/libs/
 Bin/Win32
+Bin/Win64
 Lib
-BaseToolsBuild/
\ No newline at end of file
+BaseToolsBuild/

```

Comment:
I presume this adds a missing newline?
That is an unrelated change and will result in misleading git blame/praise output.
Please break out separately or drop altogether).

Reply:
This is dropped. Thank you~

Reply:
One moment. There is something odd. Will fix that momentarily.

Reply:
Okay, this is resolved.